src/libgrate/grate-2d.c

```diff
@@ -47,8 +47,8 @@ void grate_clear(struct grate *grate)
 		return;
 	}
 
-	err = host1x_gr2d_clear(gr2d, grate->fb->back, clear->r, clear->g,
-				clear->b, clear->a);
+	err = host1x_gr2d_clear(gr2d, grate->fb->back ?: grate->fb->front,
```

Comment:
It is GCC extension, works fine in the UNIX world. https://gcc.gnu.org/onlinedocs/gcc-4.6.4/gcc/Conditionals.html